src/htm/regions/TMRegion.cpp

```diff
@@ -47,7 +47,7 @@ TMRegion::TMRegion(const ValueMap &params, Region *region)
   args_.activationThreshold = params.getScalarT<UInt32>("activationThreshold", 13);
   args_.initialPermanence = params.getScalarT<Real32>("initialPermanence", 0.21f);
   args_.connectedPermanence = params.getScalarT<Real32>("connectedPermanence", 0.50f);
-  args_.minThreshold = params.getScalarT<UInt32>("minThreshold", 10);
+  args_.minThreshold = params.getScalarT<UInt32>("minThreshold", 8);
```

Comment:
ditto, please make this correspond to what we have in TM class

Reply:
>> ditto, please make this correspond to what we have in TM class

ditto to why I am keeping them the same as in the Spec.
If I do not, the unit tests fail :)